capnpy/__init__.py

```diff
@@ -1,14 +1,16 @@
 import sys
 import pkg_resources
-from capnpy.compiler.compiler import DynamicCompiler
```

Comment:
This was needed before schema was compiled without `--no-version-check`. Currently, `__version__` can be assigned after importing `capnpy` modules, but I decided to keep it this way in case `schema.py` version check will ever get re-implemented.